SML-Sandbox/ennate.egen.solutions.sml/src/main/java/ennate/egen/solutions/sml/domain/ClassificationEngine.java

```diff
@@ -26,6 +28,7 @@ public void buildModels(ArrayList<Data> trainingData, int numberOfFields) {
 				}
 			}
 			models.add(new DataModel(temp, numberOfFields));
+			testModels.add(new Model(temp, numberOfFields));
```

Comment:
Please verify that all dataPoints in temp belong to the same classId - that piece of code should be correct but I'd appreciate a review. Essentially understand all the steps involved in making a classificationEngine - including the etl or data cleaning, data cleansing steps.